src/HttpExtractorComponent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\HttpExtractor;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Uri;
+use Keboola\Component\BaseComponent;
+use Keboola\HttpExtractor\Config\ConfigDefinition;
+use Psr\Http\Message\UriInterface;
+
+class HttpExtractorComponent extends BaseComponent
+{
+    protected function getConfigDefinitionClass(): string
```

Comment:
sakra, tyhle obě funkce mi taky unikly v procesoru, doplním

Reply:
Tak to už není náhoda. Buď je to špatné API, nebo je to málo popsané.

Reply:
postupoval jsem podle README a tam nic nebylo

Reply:
Vyřešeno v keboola/php-component#4